packages/maskbook/src/plugins/PluginUI.ts

```diff
@@ -28,12 +29,14 @@ sideEffect.then(() => {
     plugins.add(FileServicePluginDefine)
     plugins.add(ITO_PluginDefine)
     plugins.add(NFTPluginsDefine)
+    plugins.add(VoicechatPluginDefine)
```

Comment:
Please remove this line

Reply:
If I do that, I am no longer able to click on the voicechat plugin button when composing a new tweet

Reply:
I mean you're adding this plugin twice (also at line 36).

Reply:
Is `webOnly` on `Flags` not working correctly in development mode?

Reply:
hmm, I don't think so. You can print out to see the value

Reply:
Found it out. Thanks!